### PR TITLE
Issue 205 - renamed io.openshift.sti.* label to shorter io.s2i.*

### PR DESCRIPTION
**Pull request by soltysh**
Address #205.

@bparees @mfojtik PTAL

### CODE REVIEW

**Comment by mfojtik** [comment]
@soltysh I would preserve 'scripts-url' as the format of this LABEL is URL (image://, http://....)

**Comment by soltysh** [comment]
@mfojtik I'll then revert that part, but will update `--scripts` flag to `--scripts-url` for the sake of having them identical.

File: pkg/docker/docker.go [hunk 1]
```diff
@@ -18,8 +18,8 @@ const (
 	ScriptsURLEnvironment = "STI_SCRIPTS_URL"
 	LocationEnvironment   = "STI_LOCATION"
 
-	ScriptsURLLabel = "io.openshift.sti.scripts-url"
-	LocationLabel   = "io.openshift.sti.location"
+	ScriptsURLLabel = "io.s2i.scripts-url"
+	LocationLabel   = "io.s2i.location"
```
Comment (bparees) [inline pkg/docker/docker.go]: i've never liked calling this location... how about destination?  tmpdir?  target?  workdir?

**Comment by bparees** [comment]
one question about the location label name, otherwise lgtm.

**Comment by soltysh** [comment]
Destination sounds reasonable, I'll update it accordingly.

**Comment by soltysh** [comment]
@bparees renamed location -> destination. PTAL at your leisure.

[test] in the mean time.

**Comment by bparees** [comment]
lgtm.
[merge]

**Pull request merged**